### PR TITLE
add main menu and cinematics

### PR DESCRIPTION
**Pull request by krtrinkunas**

### CODE REVIEW

File: Assets/Scripts/MainMenu/MainMenuScript.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class NewBehaviourScript : MonoBehaviour
```
Comment (DreamUpstream) [inline Assets/Scripts/MainMenu/MainMenuScript.cs:5]: tuščias script

File: Assets/Scripts/MenuScript/MenuScript.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class MenuScript : MonoBehaviour
+{
+    public void PlayGame()
+    {
```
Comment (DreamUpstream) [inline Assets/Scripts/MenuScript/MenuScript.cs:9]: SceneManager.LoadScene FirstRestourant

**Review by DreamUpstream — CHANGES_REQUESTED** [review]

File: Assets/Scripts/MenuScript/MenuScript.cs (hunk 2 above)
Comment (DreamUpstream) [reply Assets/Scripts/MenuScript/MenuScript.cs:9]: is missing*

File: Assets/Scripts/MainMenu/MainMenuScript.cs (hunk 1 above)
Comment (krtrinkunas) [reply Assets/Scripts/MainMenu/MainMenuScript.cs]: removed

File: Assets/Scripts/MenuScript/MenuScript.cs (hunk 2 above)
Comment (krtrinkunas) [reply Assets/Scripts/MenuScript/MenuScript.cs]: removed, since PlayIntro method is used

**Review by DreamUpstream — APPROVED** [review]

**Pull request merged**